File: pgm/ABC_Java/ABC_327_A.java
import java.io.*;
// import java.math.*;
// import java.util.*;
// import java.util.Map.*;
// import java.util.stream.*;

public class Main {
    public static void main (String[] args) throws Exception {
        BufferedReader reader =
            new BufferedReader (new InputStreamReader (System.in));
        reader.readLine();
        final String S = reader.readLine();
        final String a = "a";
        final String b = "b";
        final String Yes = "Yes";
        final String No = "No";
        System.out.println (S.contains (a.concat (b))   ? Yes
                            : S.contains (b.concat (a)) ? Yes
                                                        : No);
        return;
    }
}

File: pgm/ABC_Kotlin/ABC_327_A.kt
// import java.io.BufferedOutputStream
// import java.io.BufferedReader
// import java.io.DataInputStream
// import java.io.FileInputStream
// import java.io.IOException
// import java.io.InputStreamReader
// import java.io.PrintWriter
// import java.lang.StringBuilder
// import java.time.*
// import java.util.StringTokenizer

// import kotlin.math.*

const val Yes: String = "Yes"
const val No: String = "No"
const val ab: String = "ab"
const val ba: String = "ba"
const val space: String = " "

fun main() {
    readln()
    val S: String = readln()
    if (S.contains(ab) || S.contains(ba)) {
        println(Yes)
    } else {
        println(No)
    }
    kotlin.system.exitProcess(0)
}